EasyPost.Tests.Net/PickupTest.cs

```diff
@@ -1,160 +1,83 @@
-                    "width", 6
-                },
-                {
-                    "height", 5
-                },
-                {
-                    "weight", 10
-                }
-            };
-            toAddress = new Dictionary<string, object>
-            {
-                {
-                    "company", "Simpler Postage Inc"
-                },
-                {
-                    "street1", "164 Townsend Street"
-                },
-                {
-                    "street2", "Unit 1"
-                },
-                {
-                    "city", "San Francisco"
-                },
-                {
-                    "state", "CA"
-                },
-                {
-                    "country", "US"
-                },
-                {
-                    "zip", "94107"
-                }
-            };
-            fromAddress = new Dictionary<string, object>
-            {
-                {
-                    "name", "Andrew Tribone"
-                },
-                {
-                    "street1", "480 Fell St"
-                },
-                {
-                    "street2", "#3"
-                },
-                {
-                    "city", "San Francisco"
-                },
-                {
-                    "state", "CA"
-                },
-                {
-                    "country", "US"
-                },
-                {
-                    "zip", "94102"
-                }
-            };
-            shipment = Shipment.Create(new Dictionary<string, object>
-            {
-                {
-                    "parcel", parcel
-                },
-                {
-                    "to_address", toAddress
-                },
-                {
-                    "from_address", fromAddress
-                },
-                {
-                    "reference", "ShipmentRef"
-                }
-            });
-            shipment.Buy(shipment.LowestRate());
-
-            parameters = new Dictionary<string, object>
-            {
-                {
-                    "is_account_address", false
-                },
-                {
-                    "instructions", "In mailbox."
-                },
-                {
-                    "address", address
-                },
-                {
-                    "shipment", shipment
-                },
-                {
-                    "min_datetime", DateTime.Now.AddDays(1)
-                },
-                {
-                    "max_datetime", DateTime.Now.AddDays(1)
-                }
-            };
+            VCR.SetUp(VCRApiKey.Test, "pickup", true);
+        }
+
+        private static Pickup CreateBasicPickup()
+        {
+            Shipment shipment = Shipment.Create(Fixture.OneCallBuyShipment);
+            Dictionary<string, object> pickupData = Fixture.BasicPickup;
+            // FIXME: If error, this might be wrong, should be data not an object
```

Comment:
Here at EasyPost we tend to use `TODO` comments only, makes it easy to quickly search for things that need attention. May help this not get missed.